src/main/java/svenhjol/charm/crafting/feature/Barrel.java

```diff
@@ -62,7 +62,7 @@ public void configure()
         useCharmBarrels = propBoolean(
             "Use Charm barrels",
             "Charm's barrels will be enabled even if barrels from other mods are present.",
-            false
+            true
```

Comment:
I will add this to the release instructions otherwise people won't see Charm's barrels alongside Future MC's.